src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java

```diff
@@ -38,55 +39,55 @@ public void whenANumberIsInACertainInterval() {
         int sum = a + b;
 
         // Check that the sum is between 10 and 20
-        assertThat(true).isFalse();
+        assertThat(sum).isBetween(10, 20);
     }
 
     @Test
     public void whenANumberIsCloseToAnotherValue() {
         double roughlyOneHundred = 99;
 
         // Check that the sum is within 10% of 100
-        assertThat(true).isFalse();
+        assertThat(roughlyOneHundred).isCloseTo(100, Percentage.withPercentage(10));
     }
 
     @Test
     public void whenAStringIsEqualToAnother() {
         String color = "red";
 
         // Check that the string has a value of 'red'
-        assertThat(true).isFalse();
+        assertThat(color).contains("red");
```

Comment:
contains works.. but is not intended i think
`assertThat("darkred").contains("red");` is also true and we are asking about equality here. so `isEqual` is probably what's asked for here, right?

Reply:
Yeah, just saw it now.  you are correct.  I'll update it.